### ISSUE TITLE
Command CoreMLModelCompile failed with a nonzero exit code

### ISSUE THREAD

**Issue by Elwin-zhao** [huggingface/swift-coreml-transformers#18]
Command CoreMLModelCompile failed with a nonzero exit code

**Comment by hollance** [comment]
You'll need to provide more info, @ITBackkom. 

Usually there is an additional error message that explains why CoreMLModelCompile failed (you can see this in the build log pane in Xcode). What does this error message say?

Which model does it fail on?

What is the file size of that model file when you look at it with Finder?

**Comment by mredig** [comment]
I'm also running into this. I think this is the error you're looking for:

`coremlc: Error: Error reading protobuf spec. validator error: unable to deserialize object`

<img width="955" alt="image" src="https://user-images.githubusercontent.com/6509485/83097243-3c31d680-a06d-11ea-9d53-cfe20bf24443.png">


Using Xcode 11.5 on macOS 10.15.3. Simply downloaded the zip and tried to build project/run tests/run project.

**Comment by hollance** [comment]
What is the file size of that model file when you look at it with Finder?

This repo uses git-lfs to store the large model files. Most likely, your model file is only a few bytes in size and is a placeholder for the actual thing.

**Comment by mredig** [comment]
> What is the file size of that model file when you look at it with Finder?
> 
> 
> 
> This repo uses git-lfs to store the large model files. Most likely, your model file is only a few bytes in size and is a placeholder for the actual thing.

I’m pretty sure this is correct. Did I miss any getting started instructions anywhere? I downloaded the zip instead of cloning, if that makes a difference at all.

**Comment by julien-c** [comment]
Yes that's definitely the reason then. Thanks @hollance for helping.

You need to install git-lfs following https://github.com/huggingface/swift-coreml-transformers#notes

**Comment by mredig** [comment]
Does that mean that downloading the zip is useless? (might warrant a readme update)

**Comment by jeffxtang** [comment]
Apple has [sample code and model](https://developer.apple.com/machine-learning/models/#text) that runs without requiring lfs.
